cli/fmt.rs

```diff
@@ -106,28 +111,38 @@ async fn check_source_files(
 
   let not_formatted_files_count =
     not_formatted_files_count.load(Ordering::SeqCst);
+  let checked_files_count = checked_files_count.load(Ordering::SeqCst);
+  let checked_files_str =
+    format!("{} {}", checked_files_count, files_str(checked_files_count));
```

Comment:
Nitpick: move into `if verbose` branch

Reply:
`checked_files_str` is used both in `if verbose` branch and in `else` branch, so I placed it here.